foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-KeyVaultPolicySetDefinition.json

```diff
@@ -546,6 +582,69 @@
                                 "value": "[[parameters('keyVaultPurgeProtection')]"
                             }
                         }
+                    },
+                    {
+                        "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/12ef42cb-9903-4e39-9c26-422d29570417",
+                        "policyDefinitionReferenceId": "Deny-KV-Cert-without-LifeTimeActionTrigger",
+                        "groupNames": [
+                            "Encryption"
+                        ],
+                        "parameters": {
+                            "effect": {
+                                "value": "[[parameters('keyVaultCertificateLifeTimeAction')]"
+                            },
+                            "minimumDaysBeforeExpiry": {
+                                "value": "[[parameters('keyVaultCertificateMinimumDaysBeforeExpiry')]"
+                            },
+                            "maximumPercentageLife": {
+                                "value": "[[parameters('keyVaultCertificateMaximumPercentageLife')]"
+                            }
+                        }
+                    },
+                    {
+                        "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/82067dbb-e53b-4e06-b631-546d197452d9",
+                        "policyDefinitionReferenceId": "Deny-KV-RSA-Keys-without-MinKeySize",
+                        "groupNames": [
+                            "Encryption"
+                        ],
+                        "parameters": {
+                            "effect": {
+                                "value": "[[parameters('keyVaultCheckMinimumRSAKeySize')]"
+                            },
+                            "minimumRSAKeySizeValue": {
```

Comment:
is this the exact parameter name? or should "value" be removed from it?

Reply:
The parameter name in the built-in definition must be referenced correctly. I.e., the name has to match and we must provide a value that is supported/in the explicit allowedValues list. This seems to be a mismatch where the policy parameter in the built-in is named "minimumRSAKeySize" and not "minimumRSAKeySizeValue".